operate/webapp/src/main/java/io/camunda/operate/webapp/security/permission/PermissionsService.java

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.operate.webapp.security.permission;
+
+import io.camunda.authentication.entity.CamundaUser;
+import io.camunda.operate.exceptions.OperateRuntimeException;
+import io.camunda.operate.property.OperateProperties;
+import io.camunda.operate.webapp.security.identity.IdentityPermission;
+import io.camunda.security.auth.Authorization;
+import io.camunda.security.auth.SecurityContext;
+import io.camunda.security.impl.AuthorizationChecker;
+import io.camunda.zeebe.protocol.record.value.AuthorizationResourceType;
+import io.camunda.zeebe.protocol.record.value.PermissionType;
+import jakarta.annotation.PostConstruct;
+import java.util.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+public class PermissionsService {
+
+  public static final String RESOURCE_KEY_ALL = "*";
+  public static final String RESOURCE_TYPE_PROCESS_DEFINITION = "process-definition";
+  public static final String RESOURCE_TYPE_DECISION_DEFINITION = "decision-definition";
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(PermissionsService.class);
+
+  private final OperateProperties operateProperties;
+  private final AuthorizationChecker authorizationChecker;
+  private final TenantsService tenantsService;
+
+  public PermissionsService(
+      final OperateProperties operateProperties,
+      final AuthorizationChecker authorizationChecker,
+      final TenantsService tenantsService) {
+    this.operateProperties = operateProperties;
+    this.authorizationChecker = authorizationChecker;
+    this.tenantsService = tenantsService;
+  }
+
+  @PostConstruct
+  public void logCreated() {
+    LOGGER.debug("PermissionsService bean created.");
+  }
```

Comment:
This log is not needed anymore as `PermissionsService` bean is not conditional anymore